zerver/lib/export.py

```diff
@@ -1497,6 +1499,23 @@ def do_import_realm(import_dir: Path) -> None:
         data = ujson.load(f)
 
     import_attachments(data)
+    return realm
+
+def do_import_system_bots(realm: Any) -> None:
+    internal_bots = [(bot['name'], bot['email_template'] % (settings.INTERNAL_BOT_DOMAIN,))
+                     for bot in settings.INTERNAL_BOTS]
+    create_users(realm, internal_bots, bot_type=UserProfile.DEFAULT_BOT)
+    names = [(settings.FEEDBACK_BOT_NAME, settings.FEEDBACK_BOT)]
+    create_users(realm, names, bot_type=UserProfile.DEFAULT_BOT)
+    print("Finished importing system bots.")
+
+def create_users(realm: Realm, name_list: Iterable[Tuple[Text, Text]], bot_type: int=None) -> None:
```

Comment:
To avoid duplication of function definition, what about eliminating the code in `./zerver/management/commands/initialize_voyager_db.py` that it imports the function from here?

Reply:
@rht But here, in the function `create_users`, the code is a little different. I am checking if the users (system_bots) already exist in the db, and if not, then only I am using bulk_create.

Reply:
I see.

Reply:
It'd be worth adding comment stating that the `create_users` and `do_import_system_bots` differ from their equivalent in `zerver/management/commands/initialize_voyager_db.py`.